java/adapter/avro/pom.xml

```diff
@@ -56,4 +56,30 @@ under the License.
       <version>${dep.avro.version}</version>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration combine.self="append">
```

Comment:
Append should mean that none of these other arguments are required, right? Only compilerArgs?

Reply:
Also should it be `combine.self` or `combine.children`?

Reply:
Yes, it seems to be the best. I updated, let's wait for the CIs to make sure.